models/intermediate/int_github__issue_joined.sql

```diff
@@ -65,30 +69,54 @@ select
   end as url_link,
   issue_open_length.days_issue_open,
   issue_open_length.number_of_times_reopened,
+
+  {%- if var('github__using_issue_label', True) and var('github__using_label', True) -%}
   labels.labels,
+  {%- else %}
+  cast(null as {{ dbt.type_string() }}) as labels,
+  {% endif -%}
+
   issue_comments.number_of_comments,
   repository_teams.repository,
-  {% if var('github__using_repo_team', true) %}
+
+  {%- if var('github__using_repo_team', true) %}
   repository_teams.repository_team_names,
-  {% endif %}
+  {% endif -%}
+
+  {%- if var('github__using_issue_assignee', True) -%}
   issue_assignees.assignees,
+  {%- else -%}
+  cast(null as {{ dbt.type_string() }}) as assignees,
+  {% endif -%}
+
   creator.login_name as creator_login_name,
   creator.name as creator_name,
   creator.company as creator_company,
+
+  {%- if var('github__using_requested_reviewer_history', True) -%}
   hours_request_review_to_first_review,
   hours_request_review_to_first_action,
   hours_request_review_to_merge,
```

Comment:
For fields like this that are no longer going to persist to the end model if the variables are disabled, we should updated the documentation accordingly to highlight that these fields will only be present with certain variables enabled.